common.py

```diff
@@ -71,6 +71,26 @@ def __init__(self, request, response):
     # Settings can't be global in python 2.7 env.
     logging.getLogger().setLevel(logging.DEBUG)
 
+  def _is_user_whitelisted(self, user):
+    if not user:
+      return False
+
+    is_whitelisted = False
+
+    if users.is_current_user_admin():
+      is_whitelisted = True
+    elif user.email().endswith('@chromium.org') or user.email().endswith('@google.com'):
```

Comment:
There are emails from other domains. I don't know if there are any unaffiliated contributors using @gmail.com. I do know there are email addresses from Intel, Samsung, Microsoft, and others.

Reply:
Right. I see it.